sql/5_0_3-to-5_0_4_upgrade.sql

```diff
@@ -0,0 +1,93 @@
+--
```

Comment:
This should be in sql/5_0_2-to-5_0_3_upgrade.sql

Reply:
ok will delete the new file